scripts/rick.js

```diff
@@ -63,6 +65,10 @@ const isRickAtPointer = (i) => {
   return distanceFromPointer < distanceFromPointerToStop;
 };
 
+const increaseRickCounter = () => {
+  rickCount++;
+  $("#rick-counter").text(rickCount.toString(2));
```

Comment:
Hey man, not everyone is fluent in binary like you... mind outputting something a bit more readable? 🤣